app/config/router-config.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = ['$stateProvider', '$urlRouterProvider', routerConfig];
+
+function routerConfig($stateProvider, $urlRouterProvider) {
+  $urlRouterProvider.when('', '/home');
```

Comment:
Good job passing in an empty string as the first arg to $urlRouterProvider, it's a common mistake to pass in '/' instead which would force the user to explicitly pass /#!/ to the url in order to change routes. Nice catch 👍